src/oca_github_bot/config.py

```diff
@@ -92,3 +92,13 @@ def func_wrapper(*args, **kwargs):
 MIN_PR_AGE = int(os.environ.get("MIN_PR_AGE", "5"))
 
 SIMPLE_INDEX_ROOT = os.environ.get("SIMPLE_INDEX_ROOT")
+
+OCABOT_USAGE = (
+    "**Ocabot commands**\n"
+    "* ``ocabot merge major|minor|patch|nobump``\n\n"
+    "**More information**\n"
+    " * [ocabot documentation](https://github.com/OCA/oca-github-bot/#commands)\n"
+    " * [OCA guidelines](https://github.com/OCA/odoo-community.org/blob/master/"
+    "website/Contribution/CONTRIBUTING.rst), "
+    'specially the "Version Numbers" section.'
+)
```

Comment:
At some point we'll probably want to make this configurable too.

Reply:
Done. 

In fact, splitted into 2 parameters. a some generic one OCABOT_USAGE, and another one OCABOT_EXTRA_DOCUMENTATION for the links, etc.